traits/util/resource.py

```diff
@@ -160,15 +160,15 @@ def find_resource(project, resource_path, alt_path=None, return_path=False):
         # script is called directly from the command line using
         # 'python %SOMEPATH%/<script>'
         if alt_path is None:
-            return
```

Comment:
This function is documented as explicitly returning `None` or a valid path; as such, the `return`s should be explicit about returning `None` (per PEP8).